crates/bevy_asset/src/handle.rs

```diff
@@ -71,6 +71,16 @@ where
     marker: PhantomData<T>,
 }
 
+// SAFE: T is phantom data and all other fields implement `Send` and `Sync`
+unsafe impl<T: Asset> Send for Handle<T> {}
+unsafe impl<T: Asset> Sync for Handle<T> {}
+
+fn _assert_send_sync_handle_impl_safe() {
```

Comment:
Could someone please explain to me what these functions are for? There is another one at `crates/bevy_ui/src/flex/mod.rs`.

Reply:
They make sure that all the fields we expect implement `Send` and `Sync` do actually implement it.

Reply:
I've used this trick elsewhere for safe auto Send/Sync impls. Any reason not to use it here?

```rust
    // NOTE: PhantomData<fn()-> T> gives this safe Send/Sync impls
    marker: PhantomData<fn() -> T>,
```

Reply:
Using it now.